.travis.yml

```diff
@@ -15,7 +15,17 @@ matrix:
     - os: linux
       rust: nightly
     - os: osx
-      osx_image: xcode11
+      osx_image: xcode11.2
+      addons:
+        chrome: stable
+        firefox: latest
```

Comment:
Do you know if there's a ticket we can check for progress on this through the release pipeline, so that we one fine day can switch to "stable" here?

Reply:
`firefox` & `chrome` have different values available:

* [Firefox](https://docs.travis-ci.com/user/firefox)
* [Chrome](https://docs.travis-ci.com/user/chrome)

IOW there's no `stable` for Firefox. But it can be pinned to the specific version, which on the other side means that one has to update it regularly.

Reply:
aha, so "latest" means "latest stable release" for FF? I guess so, given they provide "latest-dev", "latest-beta", "latest-nightly" etc... Clear as mud! :)

The I think we want "latest" here, just like you did it.